Dockerfile_binary_openvino

```diff
@@ -15,7 +15,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     python3-pip \
     python3-venv \
     usbutils \
-    virtualenv
+    virtualenv \
+    libboost-filesystem1.58.0 \
+    libboost-thread1.58.0
 
 ADD l_openvino_toolkit*.tgz $TEMP_DIR/
 RUN cd $TEMP_DIR/l_openvino_toolkit* && \
```

Comment:
OV R2 publishes binaries as apt packages. It will simplify installation is we just go with that.

Reply:
Sounds good, but I think I could apply it later on, in separate pull request.

Reply:
right, and it looks like apt is based on the open-source version, it doesn't have hddl support.